include/networkit/scd/CliqueDetect.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef NETWORKIT_SCD_CLIQUE_DETECT_HPP_
+#define NETWORKIT_SCD_CLIQUE_DETECT_HPP_
+
+#include <networkit/scd/SelectiveCommunityDetector.hpp>
+
+namespace NetworKit {
+
+/**
+ * The CliqueDetect algorithm. First finds the largest clique in the seed node's
+ * neighborhood.
+ *
+ * The algorithm can handle weighted graphs.
+ */
+class CliqueDetect : public NetworKit::SelectiveCommunityDetector {
```

Comment:
I guess that writing `NetworKit::` is redundant here since we are already in the NetworKit namespace.

Reply:
Fixed in 56c52c5639441be8bda89fbac450c6c1689c709b